.github/workflows/rust.yml

```diff
@@ -81,16 +81,6 @@ jobs:
       - uses: actions/checkout@v3
         with:
           submodules: true
-      - name: Install protobuf compiler
```

Comment:
This doesn't appear to be necessary anymore, the version in the system package manager is recent enough (only 2 years out of date) :laughing: